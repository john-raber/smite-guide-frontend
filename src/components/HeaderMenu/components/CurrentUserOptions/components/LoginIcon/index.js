import React from "react";
import { NavLink } from "reactstrap";
import { FaSignInAlt } from "react-icons/fa";

const LoginIcon = () => {
  return (
    <NavLink href="/login" className="loginIcon">
      Sign Up/Sign In <FaSignInAlt color="white" />
    </NavLink>
  );
};

export default LoginIcon;
