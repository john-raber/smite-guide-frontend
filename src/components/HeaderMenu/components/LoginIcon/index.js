import React from "react";
import { NavLink } from "reactstrap";
import { FaSignInAlt } from "react-icons/fa";

const LoginIcon = () => {
  return (
    <NavLink href="/login">
      Sign Up/Sign In <FaSignInAlt color="white" />
    </NavLink>
  );
};

export default LoginIcon;
