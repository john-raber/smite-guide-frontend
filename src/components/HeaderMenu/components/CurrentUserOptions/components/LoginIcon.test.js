import React from "react";
import { shallow } from "enzyme";

import LoginIcon from "./LoginIcon/index";

describe("LoginIcon", () => {
  it("renders without crashing", () => {
    const wrapper = shallow(<LoginIcon />);

    expect(wrapper.find(".loginIcon").length).toEqual(1);
  });

  it("matches its snapshot", () => {
    const wrapper = shallow(<LoginIcon />);

    expect(wrapper).toMatchSnapshot();
  });
});
