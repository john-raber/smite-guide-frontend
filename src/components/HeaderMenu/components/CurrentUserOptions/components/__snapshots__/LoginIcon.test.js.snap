// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoginIcon matches its snapshot 1`] = `
<NavLink
  className="loginIcon"
  href="/login"
  tag="a"
>
  Sign Up/Sign In 
  <FaSignInAlt
    color="white"
  />
</NavLink>
`;
